packages/fast-development-site-react/src/components/shell/action-bar.tsx

```diff
@@ -11,9 +11,7 @@ export interface IShellActionBarManagedClasses {
 
 const style: ComponentStyles<IShellActionBarManagedClasses, IDevSiteDesignSystem> = {
     shell__action_bar: {
-        background: (config: IDevSiteDesignSystem): string => {
-            return config.lightGrey;
-        },
+        background: "#FFFFFF",
```

Comment:
Use backgroundColor from design system?

Reply:
+1 suggest adding `backgroundColor` to the global config and updating all references to use it.

Reply:
Duh... Should have caught that. Thanks